collectd/java.sls

```diff
@@ -1,22 +1,10 @@
-{% from "collectd/map.jinja" import collectd with context %}
+{% from "collectd/map.jinja" import collectd_settings with context %}
+{% set java_settings = collectd_settings.get('plugins:java') %}
```

Comment:
I'm not sure get will work like this here. I think you'll be getting python's built-in get (which doesn't support : traversal).